db/seed_items.rb

```diff
@@ -50,311 +50,312 @@ def make_items
-# item_thirty = Item.create(
-#   name: "Apple iPhone 12 Pro Max",
-#   price: 999,
-#   link: "https://www.amazon.com.au/Apple-iPhone-12-Pro-Max/dp/B08MWLT7MS/ref=sr_1_1?crid=1T1RW3YRN0N1Q&dchild=1&keywords=iphone+12+pro+max&qid=1613796927&sprefix=iphone+12+%2Caps%2C318&sr=8-1",
-#   description: "Apple iPhone 12 Pro Max MGDL3X/A 512GB - Pacific Blue - iPhone 12 Pro. Superfast 5G. A14 Bionic, the fastest chip in a smartphone.",
-#   image: "https://images-na.ssl-images-amazon.com/images/I/61unEqDNjvL._AC_SL1100_.jpg",
-#   wishlist_id: rand(1..18)
-# )
-# user_image = URI.parse(user_image).open
-# item_one.images.attach(io: user_image, filename: "image.jpeg", content_type: 'image/jpeg')
-
-# user_image = "https://images-na.ssl-images-amazon.com/images/I/615AHkg5RqL._AC_SL1500_.jpg"
-# item_thirty_one = Item.create(
-#   name: "Grillz Charcoal BBQ Smoker Camping Grill Portable Barbeque",
-#   price: 35,
-#   link: "https://www.amazon.com.au/Grillz-Charcoal-Camping-Portable-Barbeque/dp/B07MFKL1BY/ref=zg_bs_5707524051_23?_encoding=UTF8&psc=1&refRID=YKEGJC43HAFYR6AQ1G50",
-#   description: "Looking for a quick and hassle-free BBQ? Then consider our Portable Charcoal BBQ. It is compact and lightweight.",
-#   wishlist_id: rand(1..18)
-# )
-# user_image = URI.parse(user_image).open
-# item_one.images.attach(io: user_image, filename: "image.jpeg", content_type: 'image/jpeg')
-
-# user_image = "https://images-na.ssl-images-amazon.com/images/I/61BtkSjcC4L._AC_SL1000_.jpg"
-# item_twenty_two = Item.create(
-#   name: "Automatic Pet Water Fountain",
-#   price: 40,
-#   link: "https://www.amazon.com.au/LEMONDA-Automatic-Transparent-Dispenser-Waterfall/dp/B089RCN1Q1/ref=zg_bs_5581859051_22?_encoding=UTF8&psc=1&refRID=CYGHDPXB2S2XP1CQMK5W",
-#   description: "The largest capacity design on the market which can meet the pet's need for several days,perfect for dogs , cats , multiple pet households.",
-#   wishlist_id: rand(1..18)
-# )
-# user_image = URI.parse(user_image).open
-# item_one.images.attach(io: user_image, filename: "image.jpeg", content_type: 'image/jpeg')
+  item_five.images.attach(io: user_image, filename: "image.jpg", content_type: 'image/jpg')
+
+
+user_image = "https://images-na.ssl-images-amazon.com/images/I/513YGV2RT4L._AC_SL1000_.jpg"
+item_six = Item.create(
+  name: "Echo Wall Clock",
+  price: 49.00,
+  link: "https://www.amazon.com.au/Echo-Wall-Clock-See-timers-at-a-glance/dp/B07SP78TFY/?_encoding=UTF8&pd_rd_w=CaOWS&pf_rd_p=d2e5f020-22cd-4dfd-8518-73f1e2eddd6f&pf_rd_r=8W65MCAT12YAV8R6WJKF&pd_rd_r=6f186116-6c39-4d58-a316-53c3a032f147&pd_rd_wg=cn47x&ref_=pd_gw_trq_rep_sims_gw",
+  description: "Echo Wall Clock helps you stay organised and be on time. Easy-to-read analogue clock shows the time.",
+  wishlist_id: rand(1..18)
+)
+user_image = URI.parse(user_image).open
+item_one.images.attach(io: user_image, filename: "image.jpg", content_type: 'image/jpg')
```

Comment:
line065 these all need to match their partner from line #57 
This will just keep refreshing the image attached to item_one 

the 5 examples at the top of this file are matched like that, i just didn't get time to to do the whole thing :)

Reply:
Tried this the other night and it wasn't taking all of the items from the method in the seed_items.

Reply:
cool i'll look at this now and come back

Reply:
this might have been sorted in pull # 24

Reply:
Discard changes and dismiss this branch then?